Makefile.am

```diff
@@ -58,8 +57,11 @@ ChangeLog: $(srcdir)/tools/git2cl
 		$(srcdir)/tools/git2cl > $@ \
 	; fi
 
+indent:
+	cd src && $(MAKE) $(AM_MAKEFLAGS) indent
```

Comment:
And by the way, there won't exist $(top_srcdir)/src/Makefile in vpath build. It's actually $(top_builddir)/src/Makefile that is run.